source/manual/howto-switch-default-branch.html.md

```diff
@@ -0,0 +1,26 @@
+---
+owner_slack: "#govuk-developers"
+title: How to switch a git default branch from master to main
+section: GitHub
+layout: manual_layout
+parent: "/manual.html"
+---
+
+These steps are for switching the default branch of a repo to `main`.
+
+1. Rename the `master` branch in GitHub to `main` for that repo. See [instructions from Github for renaming an existing default branch](https://github.com/github/renaming#renaming-existing-branches).
+1. Update the [Release app](https://release.publishing.service.gov.uk/applications/):
+  - Go to the app entry from the [listing](https://release.publishing.service.gov.uk/applications/).
+  - Hit the `Edit` tab.
+  - Switch under the dropdown for `GitHub repository default branch`.
+1. [Reset the Jenkins build number](https://docs.publishing.service.gov.uk/manual/test-and-build-a-project-on-jenkins-ci.html#fixing-the-build-number).
```

Comment:
We might want to be explicit here on where to get the build number from, i.e. it has to be the latest `master` build number plus 1.

Reply:
Great suggestion :+1:

Reply:
I think you may have added `(the last `master` build +1)` to the wrong line, I think it should be this one rather than the one above.

Reply:
🤦